lambda.py

```diff
@@ -1,116 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This file is being removed as a new one was created called `refusereminder.py` to have a more descriptive name.